File: 10 - Python/02 - Estrutura Condicional/Exercício 07.py
#Exercício - Dardo

print("Digite as três distâncias:")
a = float(input(""))
b = float(input(""))
c = float(input(""))

if a > b and a > c:
    maior = a
elif b > c:
    maior = b
else:
    maior = c

print("Maior distância = ", maior)

File: 07 - C++/Exercícios/02 - Estrutura Condicional/Exercício 07.cpp
//Exercício - Dardo

#include <iostream>

using namespace std;

int main() {

    double a, b, c, maior;

    cout << "Digite as tres distancias:\n";
    cin >> a;
    cin >> b;
    cin >> c;

    if (a > b && a > c){
        maior = a;
    }
    else if(b > c){
        maior = b;
    }
    else{
        maior = c;
    }

    cout << "Maior distancia = " << maior << endl;

    return 0;

}